### DISCUSSION TITLE
posts/subredes/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#17] [General]
# posts/subredes/

¿Qué son las subredes?

https://marcosruiz.github.io/posts/subredes/

**Comment by amosISA** [comment]
Una web que me parece muy completa para el tema del cálculo de subredes es la [calculadora ip](https://calculadoraip.org/) que contiene cálculos respecto de FLSM, VLSM, supernetting, ipv6, grandes redes y muchos más contenido interesantes...

**Comment by AtharvM02222** [comment]
https://forms.office.com/Pages/ResponsePage.aspx?id=dqBGjZPQbUGle4aSzeE7-PCGp7AtWyFOqaCnOD-k8SFUMVhTVFVWTFUxSlBJSjM2RklMVkNIS0w3OS4u
